server/grpcservice/util.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package grpcservice
+
+import (
+	"context"
+	"crypto/tls"
+	"net/url"
+
+	"github.com/CeresDB/ceresdbproto/pkg/commonpb"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// GetClientConn returns a gRPC client connection.
+func GetClientConn(ctx context.Context, addr string, tlsCfg *tls.Config, do ...grpc.DialOption) (*grpc.ClientConn, error) {
+	var creds credentials.TransportCredentials
```

Comment:
Now the tls is not necessary.

Reply:
grpc.WithInsecure() is  deprecated. In addition, this is a util func, so i expose tlsCfg.